install.sh

```diff
@@ -100,6 +100,29 @@ check_platform() {
   return 0
 }
 
+setup_version_file() {
```

Comment:
This would not work in the pypi install, rpm install, or any install that does not use the install.sh.

Reply:
Could we simply use a `setup.py.in` with a placeholder for the version, i.e. `version=@VERSION@`, which we can replace at build-time? This could then be used by all means to install ramalama by simply adding this prepare step. Additionally, the version would be hard-coded. We use that approach for example in [BlueChi](https://github.com/eclipse-bluechi/bluechi/blob/main/src/bindings/python/setup.py.in#L15)
WDYT? @dougsland @rhatdan

Reply:
@engelmi I like the idea but maybe we will close this one. WDYT @rhatdan @engelmi  ?